<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Profile:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="profileSelector">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="profileRenameButton">
         <property name="icon">
          <iconset resource="../icons/collection.qrc">
           <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="profileDeleteButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../icons/collection.qrc">
           <normaloff>:/icons/trash.svg</normaloff>:/icons/trash.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="repoTabSlot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Repository</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="sourceTabSlot">
        <attribute name="title">
         <string>Sources</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="scheduleTabSlot">
        <attribute name="title">
         <string>Schedule</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="archiveTabSlot">
        <attribute name="title">
         <string>Archives</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="miscTabSlot">
        <attribute name="title">
         <string>Misc</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1" alignment="Qt::AlignTop">
        <widget class="QLabel" name="createProgressText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignTop">
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="createProgress">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>151</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>151</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>220</red>
               <green>220</green>
               <blue>220</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="createStartBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Start Backup</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLatest">
   <property name="text">
    <string>Latest</string>
   </property>
  </action>
  <action name="actionResetApp">
   <property name="text">
    <string>Reset App</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/collection.qrc"/>
 </resources>
 <connections/>
</ui>
